hypothesis-python/src/hypothesis/internal/scrutineer.py

```diff
@@ -114,6 +114,9 @@ def __exit__(self, *args, **kwargs):
     f"{sep}_pytest{sep}assertion{sep}rewrite.py",
     f"{sep}_pytest{sep}_io{sep}saferepr.py",
     f"{sep}pluggy{sep}_result.py",
+    # These are triggered by gc callbacks, for some reason
+    f"{sep}reprlib.py",
+    f"{sep}_pytest{sep}assertion{sep}util.py",
```

Comment:
This makes the tests pass (fingers crossed), but is it just papering over something that should be fixed?

Reply:
Note, the failures can be reproduced on master by adding
```
import gc; gc.callbacks.append(lambda *_: None)
```
at top level of conftest.py, so it's nothing to do with the other code changes.

Reply:
Nah, this is totally fine - the whole purpose of this list is to suppress reports of unhelpful locations, and stdlib or pytest internals are almost always in that category.  (at some point maybe we should generalize this further with an is_pytest_file helper?)

Reply:
Would you support something like this: https://github.com/jobh/hypothesis/commit/f2a633c04db5058889bd103a3d37889b31fa1102

It's just adding some glob wildcards